pages/open_cas_linux/guide_tool_details.md

```diff
@@ -758,6 +759,12 @@ Threshold is expressed in KiB.
 **[-j, -\-core-id \<ID\>]**: Unique identifier for core \<0 to 4095\>. If not
 specified, core parameter is set to all cores in a given cache.
 
+**[-\-promotion-count \<NUMBER\>]**: Specify how many times IO stream from a
+single thread has to be requested before it is rescheduled to another thread.
```

Comment:
Specify how many requests an IO stream from a single thread is expected to be produce before this thread is rescheduled to another CPU.